projects/driver_display/inc/driver_display_brightness.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+// Module used to control the brightness of the driver display screens.
+// Also defines display screen parameters such as frequency and number of screens
+
+#include "adc.h"
+#include "gpio.h"
+#include "pwm.h"
+#include "soft_timer.h"
+
+#define DRIVER_DISPLAY_CONFIG_NUM_SCREENS 2
```

Comment:
This shouldn't be defined here. If your plan is to fix the number of supported screens per instance, then it should be in your enum.